### PR TITLE
release notes

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: blog-service/2023-04-28-apps.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - apps
+  - app catalog
+  - saas cloud
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're excited to announce the release of the new Asana App for Sumo Logic. This app leverages Sumo Logic's [Cloud-to-Cloud Asana source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/asana-source) to provide real-time monitoring and security insight for your Asana account. Here are some of the key features that this app offers:
```
Comment (JainM6) [inline blog-service/2023-04-28-apps.md:14]:
```suggestion
We're excited to announce the release of the new Asana app for Sumo Logic. This app leverages Sumo Logic's [Cloud-to-Cloud Asana source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/asana-source) to provide real-time monitoring and security insight for your Asana account. Here are some of the key features that this app offers:
```

File: blog-service/2023-04-28-apps.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+* **Customizable searches**. Build custom searches to examine particular Asana data elements, allowing you to go deeper into the data and find insights that are pertinent to your company's needs.
```
Comment (JainM6) [inline blog-service/2023-04-28-apps.md:16]: Id' rephrase this as:

**Integration with other sources**.  Combine with other Sumo Logic apps to gain a more holistic view of the data.

**Review by JainM6 — CHANGES_REQUESTED** [review]
Formatting

File: blog-service/2023-04-28-apps.md (hunk 2 above)
Comment (JV0812) [reply blog-service/2023-04-28-apps.md:16]: updated.

**Review by JainM6 — CHANGES_REQUESTED** [review]
There is a broken link.

File: blog-service/2023-04-28-apps.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title: April 21, 2023 (Apps)
```
Comment (kimsauce) [inline blog-service/2023-04-28-apps.md:2]: wrong date

**Review by kimsauce — DISMISSED** [review]
fix date and the link Megha pointed out

**Review by JainM6 — CHANGES_REQUESTED** [review]
You need to put today's date in the filename.

File: blog-service/2023-04-28-apps.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+title: April 27, 2023 (Apps)
```
Comment (kimsauce) [inline blog-service/2023-04-28-apps.md:2]: wrong date

Comment (JV0812) [reply blog-service/2023-04-28-apps.md]: since the file will be released today, I changed file name and the date in the title as: April 28, 2023

**Review by JainM6 — APPROVED** [review]
LTGM
